packages/reactor-browser/src/actions/document.ts

```diff
@@ -149,14 +149,7 @@ export async function exportFile(document: PHDocument, suggestedName?: string) {
   if (!reactor) {
     return;
   }
-  const documentModelModules = reactor.getDocumentModelModules();
-  const documentModelModule = documentModelModules.find(
-    (module) => module.documentModel.id === document.header.documentType,
-  );
-  const extension = documentModelModule?.documentModel.extension;
-  const name = `${suggestedName || document.header.name || "Untitled"}.${
-    extension ? `${extension}.` : ""
-  }zip`;
+  const name = `${suggestedName || document.header.name || "Untitled"}.phd`;
```

Comment:
Was this a requirement? Are we no longer making use of the extension field in the document model specification?

Reply:
We never actually used the document extension. If you defined something like `.phdm`, what you ended up with was `my-file.phdm.zip`.  
I think I can keep the same behavior and just replace `.zip` with `.phd`.  
I'll push a new commit with thi